src/wp-includes/script-loader.php

```diff
@@ -1856,12 +1856,16 @@ function wp_print_delayed_inline_script_loader() {
 
 	if ( $wp_scripts->has_delayed_inline_script() ) {
 		$output    = <<<JS
-function wpLoadAfterScripts( handle ) {
-	const scripts = document.querySelectorAll(`[type="text/template"][data-wp-executes-after="\${handle}"]`);
-	scripts.forEach( (script) => {
-		script.setAttribute("type","text/javascript");
-		eval(script.textContent);
-	})
+function wpLoadAfterScripts(handle) {
```

Comment:
```suggestion
function wpLoadAfterScripts( handle ) {
```

Reply:
Resolved in recent commit @joemcgill

Reply:
Oh great! And you addressed additional spacing issues that I didn't even notice. Thanks @10upsimon!